VMBackup/main/fsfreezer.py

```diff
@@ -156,15 +157,21 @@ def freeze_safe(self,timeout):
                             self.logger.log(line.rstrip(), True)
                         else:
                             break
-                error_msg="freeze failed for some mount"
-                freeze_result.errors.append(error_msg)
-                self.logger.log(error_msg, True, 'Error')
+                if (sig_handle == 0):
```

Comment:
Suggestion : please add log of "sig_handle" here as well as we are seeing some mis-matches
i.e. inside "if(sig_handle != 1):"

Reply:
Added